.circleci/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/tsickle
+    docker:
+      - image: alexeagle/ngcontainer
+    steps:
+      - checkout
+      - restore_cache:
+          key: tsickle-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
sha1sum instead of checksum (?)
I think checksums aren't intended to be used for for content id